### PR TITLE
Feature/implement friending players

### PR DESCRIPTION
**Pull request by tysnj**
## Is this a fix or a feature?
Feature

## What is the change?
Link the playerList feature to the backend.

## What does it fix?
The playerList feature was working on front end only.

## Where should the reviewer start?
Fetch/Checkout `feature/implement_friending_players`

## How should this be tested?
Check the friendless edge case message, add and remove friends from both Friends and Community views, check form for friends. Refresh at some point to confirm data persists

### CODE REVIEW

File: src/components/EventForm/EventForm.js [hunk 1]
```diff
@@ -203,6 +203,14 @@ function EventForm({ courses, friends, hostId, refreshEvents }) {
...
                 <>
                   <p className='friends-title'>Friends to Invite:</p>
+                  { !friends.length && 
+                    <div>
+                      <em className='friendless'>
+                        You don't have any friends...<br/>
+                        Make some by creating a public event!
+                      </em>
+                    </div>
+                  }
```
Comment (jon-schlandt) [inline src/components/EventForm/EventForm.js:213]: Good thinking adding this error handling! Suggesting they create a public event is a nice touch!

File: src/components/PlayerList/PlayerList.js [hunk 2]
```diff
@@ -33,6 +33,11 @@ const PlayerList = ({screenWidth, players, friends, handleFriends}) => {
         </button>
       </span>
       <ul className='player-list-wrapper'>
+        {!friends.length && playerType === 'friends' &&
+        <em>
+            You don't have any friends...<br/>
+            Add some from the community!
+        </em>}
```
Comment (jon-schlandt) [inline src/components/PlayerList/PlayerList.js:40]: More error handling - great!

**Review by jon-schlandt — APPROVED** [review]
Looks great - glad that we were able to get this implemented!

File: src/APICalls/APICalls.js [hunk 3]
```diff
@@ -91,4 +93,40 @@ export const deleteEvent = (eventId, playerId) => {
...
+    method: 'DELETE',
+    body: JSON.stringify({
+      follower_id: followerId,
+      followee_id: followeeId
+    }),
+    headers: { 'Content-Type' : 'application/json' } 
+  })
+  .then(resp => {
+    if (resp.ok) {
+      return resp
```
Comment (errabun) [inline src/APICalls/APICalls.js:125]: Stoked we got this all implemented...would've felt like a waste a couple days if we just had to delete all that code, nice!

**Review by errabun — APPROVED** [review]

**Pull request merged**